jose/encrypt.go

```diff
@@ -40,22 +58,35 @@ func EncryptJWK(jwk *JSONWebKey, passphrase []byte) (*JSONWebEncryption, error)
 		PBES2Salt:  salt,
 	}
 
-	opts := new(EncrypterOptions)
-	opts.WithContentType(ContentType("jwk+json"))
+	encrypterOptions := new(EncrypterOptions)
+	if ctx.contentType != "" {
+		encrypterOptions.WithContentType(ContentType(ctx.contentType))
+	}
 
-	encrypter, err := NewEncrypter(DefaultEncAlgorithm, recipient, opts)
+	encrypter, err := NewEncrypter(DefaultEncAlgorithm, recipient, encrypterOptions)
```

Comment:
Is there any benefit of using AES with a 256 bit key here when that key is encrypted with a 128 bit key?

Reply:
Probably not, but that has been the default behavior for a long time, take a look at any JWK provisioner:

```shell
$ echo <encrypted-key> | step crypto jose format | jq -r .protected | step base64 -d | jq
{
  "alg": "PBES2-HS256+A128KW",
  "cty": "jwk+json",
  "enc": "A256GCM",
  "p2c": 100000,
  "p2s": "<16-byte-salt>"
}
```

The same applies to the default encryption algorithm when you use `step crypto jwe encrypt` with any password-base algorithm.

If we want to make any change would be to `PBES2-HS512+A256KW` by default. In any case, the key with more risk is actually the password.